### DISCUSSION TITLE
Use of LoHA possible?

### DISCUSSION THREAD

**Discussion by Darkflo264** [vladmandic/sdnext#283] [Q&A]
Hi everyone!

Your fork seems fantastic and interesting. I’m looking forward to give it a try this evening. 
before that I am curious if LoHAs can be used the same way as LoRAs?

thanks and best regards

**Comment by vladmandic** [comment]
similar, but not identical. place models in `models/LyCORIS` and load using <lyco:name:strenght> so app knows which network to use to interpret the model.
